src/lbpDashboard/projectTokenInfo/projectTokenInfo.html

```diff
@@ -10,7 +10,7 @@
       $ <formatted-number thousands-separated value.bind="currentPrice"></formatted-number>
     </div>
     <div class="timeleft">
-      <time-left launch.to-view="lbpMgr"></time-left>
+      <time-left launch.to-view="lbpMgr" largest></time-left>
     </div>
```

Comment:
Thread for Responsiveness check
Not always, but only for longer days

@dkent600 
Options I see here is
1. Add non-breaking-white-space between "$DIGITS $TIME", so eg. "6 hours" is broken up together (instead of "6" then "hours"), but that requires a bit more code changes in `DateService.ts`.



![image](https://user-images.githubusercontent.com/30693990/146975712-6e546de0-db37-4cd1-8918-ffd5fd1dfc57.png)

Reply:
Would it be unreasonably complicated to redefine the new "largest" as "largest2" and let Time Remaining be just largest in the original sense, only showing the largest unit, while changing its config to show seconds when < one minute?

Reply:
> redefine
I see a clash of requirements here then.
The original issue asked for adding minutes to hours (ie. largest2).
Your proposal asks to keep the "largest" unit.

cc @BartuCulha